README.md

```diff
@@ -248,6 +248,33 @@ The framework is compatible with all major desktop and mobile browsers.
 cargo run --release --bin kaspad -- --testnet
   ```
 
+<details>
+
+  <summary>
+Use a configuration file
+  </summary>
+
+  ```bash
+cargo run --release --bin kaspad -- --configfile /path/to/configfile.toml
+# or
+cargo run --release --bin kaspad -- -C /path/to/configfile.toml
+  ```
+  The config file should be a list of \<CLI argument\> = \<value\> seperated by newlines.

```

Comment:
Minor comments:

"seperated by newlines. for example:" -> "sep**a**rated by newlines. **F**or example:"